addons/binding/org.openhab.binding.kostalinverter/ESH-INF/thing/thing-types.xml

```diff
@@ -49,13 +49,1516 @@
 		</config-description>
 	</thing-type>
 	
+	<thing-type id="PIKOIQ42"> 
+			<label>KOSTAL PIKO IQ 4.2</label>
+			<description>Bindings for the KOSTAL PIKO IQ 4.2 solar inverter</description>
+			<category>Inverter</category>
```

Comment:
I found this category at https://www.eclipse.org/smarthome/documentation/concepts/categories.html
The Inverter is part of the list and from my understanding, since OH2 is based on ESH, this should be valid. 
If I missunderstood something, please let me know :)

Reply:
You right :+1: I was looking at the list of channels. This is ok.